core/domain/wipeout_service.py

```diff
@@ -590,11 +693,18 @@ def _pseudonymize_activity_models_with_associated_rights_models(
             The commit log model class.
         commit_log_model_field_name: str. The name of the field holding the
             activity id in the corresponding commit log model.
+        allowed_commands: list(dict). The commands that are allowed for the
+            activity commits.
+        rights_user_id_fields: list(str). The names of user ID fields of
+            the activity rights model.
     """
     user_id = pending_deletion_request.user_id
+    metadata_model_classes = (
+        snapshot_metadata_model_class,
+        rights_snapshot_metadata_model_class)
 
     snapshot_metadata_models, commit_log_models = (
-        _collect_activity_ids_from_snapshots_and_commit(
+        _collect_and_save_entity_ids_from_snapshots_and_commit(
```

Comment:
I'm not sure, but should this be "snapshots and commits" (plural)?

Reply:
Done